src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx

```diff
@@ -187,6 +193,14 @@ const MintTokenMotion = ({
             actionData={colonyAction}
             colony={colony}
           />
+          {currentUserName && !ethereal && (
+            <div ref={bottomElementRef}>
```

Comment:
In most cases this wouldn't be needed, but you're lucky that in this case we do :)

Reply:
Lucky me :)